build/yaml/jobs/codeanalysis/apiscan.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Job Template: Run APIScan Code Analysis
+# Invokes APIScan analysis
+
+jobs:
+- job: CodeAnaysis_APIScan
+  displayName: Code Analysis - APIScan
+
+  timeoutInMinutes: 360
+
+  # No dependencies. Requires Build Stages to ensure that binaries are built
+
+  pool:
+    name: VSEng-MicroBuildVS2019
+
+  workspace:
+    clean: outputs
+
+  steps:
+  - checkout: none
+
+  - task: DownloadBuildArtifacts@0
+    displayName: Download Windows Files (Release)
+    inputs:
+      downloadType: single
+      artifactName: binaries-windows-Release
+      downloadPath: $(Build.ArtifactStagingDirectory)
+
+  # Run API Scan
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@1
+    displayName: 'Run APIScan'
+    inputs:
+      softwareFolder: '$(Build.ArtifactStagingDirectory)/binaries-windows-Release'
+      softwareName: Aerobee
+      softwareVersionNum: '$(Build.BuildNumber)'
+      symbolsFolder: 'SRV*http://symweb;$(Build.ArtifactStagingDirectory)/binaries-windows-Release'
```

Comment:
>http://symweb [](start = 26, length = 13)

This might itself fail scans because it doesn't use https. It is an internal Microsoft website, only available in our network. Maybe consider making this a build variable?

Reply:
Static analysis builds will only be run on internal VSEng-MicroBuildVS2019 pool since they rely on internal tasks. 

---
In reply to: [387305287](https://github.com/microsoft/CLRInstrumentationEngine/pull/217#discussion_r387305287) [](ancestors = 387305287)

Reply:
I can move the URL to https

---
In reply to: [387309275](https://github.com/microsoft/CLRInstrumentationEngine/pull/217#discussion_r387309275) [](ancestors = 387309275,387305287)

Reply:
Https results in ERROR: System.ArgumentException: One or more symbol path parts are incorrect. I'll leave it as Http for now